doc/source/examples/extended_examples/sfem/stochastic_fem.rst

```diff
@@ -0,0 +1,439 @@
+the stochastic finite element method (SFEM) was introduced.
+
+The stochastic finite element method (SFEM) extends the classical deterministic finite element approach
+to a stochastic framework, offering various techniques for calculating response variability. Among these,
+the Monte Carlo simulation (MCS) stands out as the most prominent method. Renowned for its versatility and
+ease of implementation, MCS can be applied to virtually any type of problem in stochastic analysis.
+
+Random variables vs stochastic processes
+----------------------------------------
+A distinction between random variables and stochastic processes is attempted in this section. Explaining these
+concepts is important since they are used for modelling the system randomness. Random variables are easier to
+understand from elementary probability theory, the same cannot be said for stochastic processes. Readers are
+advised to consult books on SFEM if the explanation here seems too brief. [1]_ and [2]_ are recommended resources.
+
+Random variables
+~~~~~~~~~~~~~~~~
+**Definition:** A random variable is a rule for assigning to every possible outcome :math:`\theta` of an experiment a
+number :math:`X(\theta)`. For notational convenience, the dependence on :math:`\theta` is usually dropped and the
+random variable is written as :math:`X`.
+
+Practical example
++++++++++++++++++
+Imagine a beam with a concentrated load :math:`P` applied at a specific point. The value of :math:`P`
+is uncertain—it could vary due to manufacturing tolerances, loading conditions, or measurement errors. Mathematically,
+:math:`P` is a random variable:
+
+.. math:: P : \Theta \longrightarrow \mathbb{R}
+
+where :math:`\Theta` is the sample space of all possible loading scenarios, and :math:`\mathbb{R}` represents the set of
+possible load magnitudes. For example, :math:`P` could be modeled as a random variable with a probability density
+function (PDF) such as:
+
+.. math:: f_P(p) = \frac{1}{\sqrt{2\pi\sigma^2}}e^{-\frac{(p-\mu)^2}{2\sigma^2}},
+
+where :math:`\mu` is the mean load, and :math:`\sigma^2` is its variance.
+
+Stochastic processes
+~~~~~~~~~~~~~~~~~~~~
+**Definition:**
+recall that a random variable is defined as a rule that assigns a number :math:`X` to every outcome :math:`\theta`
+of an experiment. However, in some applications, the experiment evolves with respect to a deterministic parameter :math:`t`,
+which belongs to an interval :math:`I`. For example, this occurs in an engineering system subjected to random dynamic loads
+over a time interval :math:`I \subseteq \mathbb{R}^+`. In such cases, the system's response at a specific material point is
+described not by a single random variable but by a collection of random variables :math:`\{X(t)\}` indexed by :math:`t \in I`. 
+This 'infinite' collection of random variables over the interval :math:`I` is called a stochastic process and is denoted as
```

Comment:
```suggestion

This 'infinite' collection of random variables over the interval :math:`I` is called a stochastic process and is denoted as
```
Suggesting a paragraph break here as is too much text for one paragraph for online content.